src/pub/whitelist_ips.js

```diff
@@ -33,9 +34,9 @@ export default class Whitelist_ips extends Pure_component {
 }
 
 const Admin_steps = ({ip})=>{
-    const port = location.port||80;
```

Comment:
@barakluminati here, when no has port, and the protocol is ssl, the deafault port is 443 and no 80